ipgeo/leo.go

```
@@ -30,83 +31,121 @@ var IPPools = IPPool{
+	if res.Get("domain").String() == "" {
+		domain = res.Get("owner").String()
+	}
 
-		m := make(map[string][]string)
-		err := json.Unmarshal([]byte(res.Get("router").String()), &m)
-		if err != nil {
-			// 此处是正常的，因为有些IP没有路由信息
-		}
+	m := make(map[string][]string)
+	err := json.Unmarshal([]byte(res.Get("router").String()), &m)
+	if err != nil {
```

golangci-lint (2.11.4)

[error] SA9003: empty branch

(staticcheck)